README.md

```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
 
 ##### Running a Single Test
 
-If you haven't already done so, you will need to configure the HCA CLI tool to use the EBI installation of the upload service rather than the DCP (default) version.
+To run a single test, make sure that all necessary environment variables are provided.
 
 ```
-./setup_ingest_config.sh
-```
-
-To run a single test, make sure that all necessary environment variables are provided. At the time of writing, the most
-commonly required variables are `DEPLOYMENT_ENV` and `GOOGLE_APPLICATION_CREDENTIALS`.
-
-```
-export AWS_PROFILE=embl-ebi \
+export AWS_PROFILE=embl-ebi; \
 export DEPLOYMENT_ENV=dev; \
+export INGEST_API_JWT_AUDIENCE=https://dev.data.humancellatlas.org/; \
+export DEPLOYMENT_STAGE=${DEPLOYMENT_ENV}; \
+export ARCHIVER_API_KEY=$(aws secretsmanager get-secret-value --region us-east-1 --secret-id ingest/${DEPLOYMENT_ENV}/secrets --query SecretString --output text | jq -jr '.archiver_api_key'); \
+export HCA_UTIL_ADMIN_ACCESS=$(aws secretsmanager get-secret-value --region us-east-1 --secret-id hca/util/aws-access-keys --query SecretString --output text | jq -jr '.ADMIN_ACCESS_KEY'); \
+export HCA_UTIL_ADMIN_SECRET=$(aws secretsmanager get-secret-value --region us-east-1 --secret-id hca/util/aws-access-keys --query SecretString --output text | jq -jr '.ADMIN_SECRET_ACCESS_KEY'); \
+export HCA_UTIL_ADMIN_PROFILE='test-hca-util-admin'; \
```

Comment:
Is the existence of this profile a prerequisite to running the tests?

Reply:
No, it's not. The profile is being set up thru hca-util and is being done in the test. Please see https://github.com/ebi-ait/ingest-integration-tests/blob/dev/tests/runners/submission_manager.py#L40-L44